### PR TITLE
[17.0][MIG] web_editor_class_selector: Migration to version 17.0

### PR DESCRIPTION
**Pull request by carlos-lopez-tecnativa**
TT51657
@Tecnativa @pedrobaeza @chienandalu could you please review this

### CODE REVIEW

File: web_editor_class_selector/README.rst [hunk 1]
```diff
@@ -17,19 +17,21 @@ Web editor class selector
     :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
     :alt: License: AGPL-3
 .. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fweb-lightgray.png?logo=github
-    :target: https://github.com/OCA/web/tree/16.0/web_editor_class_selector
+    :target: https://github.com/OCA/web/tree/17.0/web_editor_class_selector
```
Comment (pedrobaeza) [inline web_editor_class_selector/README.rst:20]: This change shouldn't be in this commit, but on the migration one.

**Review by pedrobaeza — APPROVED** [review]
Tested on base runboat installing the module, and working. There's any module in the repo that is breaking the styles popup toolbar, so it's not possible to be tested on regular runboat.

File: web_editor_class_selector/README.rst (hunk 1 above)
Comment (carlos-lopez-tecnativa) [reply web_editor_class_selector/README.rst:20]: @pedrobaeza Sorry, but I don't understand. These changes were made automatically by the pre-commit hook, prior to the version change in the manifest.

image

Comment (pedrobaeza) [reply web_editor_class_selector/README.rst:20]: OK, my bad, sorry.

File: web_editor_class_selector/static/src/js/backend/html_field.esm.js [hunk 2]
```diff
@@ -5,9 +5,9 @@ import {useService} from "@web/core/utils/hooks";
 
 const {onWillStart} = owl;
 
-patch(HtmlField.prototype, "web_editor_class_selector.HtmlField", {
+patch(HtmlField.prototype, {
     setup() {
-        this._super(...arguments);
+        super.setup();
```
Comment (chienandalu) [inline web_editor_class_selector/static/src/js/backend/html_field.esm.js]: Maybe passing `...arguments` is a sane default practice...
```suggestion
        super.setup(...arguments);

**Review by chienandalu — APPROVED** [review]
Test and code review. 

Non really blocking comment:

**Review by chienandalu — APPROVED** [review]

**Pull request merged**